### PR TITLE
Add Redirect if Path on Chimp & See Load

### PR DESCRIPTION
**Pull request by wgranger**
This PR adds a redirect from chimpandsee.org/#/about/authors to https://www.zooniverse.org/projects/sassydumbledore/chimp-and-see/about/team on page load.

### CODE REVIEW

**Comment by zwolf** [comment]
Might want to get an FE friend to look but all I am curious about is if there's a reason to not use `$(document).ready()` and avoid the JS in the HTML?

**Comment by wgranger** [comment]
That's a good idea. I did want to avoid using JQuery, but I think it's already loading.

**Review by srallen — APPROVED** [review]
LGTM

**Comment by srallen** [comment]
Non-blocking, but if you want to avoid jquery and also avoid using inline javascript in the html, you could put in a script tag in the head and use `window.onload = redirectFunction` instead.

**Comment by wgranger** [comment]
I've removed the inline JS. I also had to make a couple changes so the function would be visible.

File: ChimpAndSee/index.html [hunk 1]
```diff
@@ -22,8 +22,13 @@
         <link rel="icon" href="favicon.ico" type="image/ico"/>
 
         <script src="js/vendor/modernizr-2.8.3-respond-1.4.2.min.js"></script>
+        <script src="js/main.js"></script>
+
+        <script>
+          window.onload = checkPath;
```
Comment (srallen) [inline ChimpAndSee/index.html:28]: I think you'll have to move the checkPath function into this script tag too or else this will be undefined.

Comment (eatyourgreens) [reply ChimpAndSee/index.html:28]: That shouldn't be a problem. `checkPath` here is a global variable.

Comment (srallen) [reply ChimpAndSee/index.html:28]: AH, I missed that main.js is loading before this.

File: ChimpAndSee/js/main.js [hunk 2]
```diff
@@ -1 +1,9 @@
+const checkPath = function() {
```
Comment (eatyourgreens) [inline ChimpAndSee/js/main.js]:
```suggestion
function checkPath() {
```
Minor thing, but declaring `checkPath` as a function, not a variable, allows you to take advantage of [hoisting](https://developer.mozilla.org/en-US/docs/Glossary/Hoisting).

File: ChimpAndSee/index.html (hunk 1 above)
Comment (eatyourgreens) [reply ChimpAndSee/index.html:28]: If it's declared as a function in main.js, then it'll be defined at runtime, even if it's declared after this script tag.

**Comment by wgranger** [comment]
This should be good to go

**Pull request merged**